frontend-ssr/services/Utils.ts

```diff
@@ -1 +1,26 @@
 export { v4 as uuid } from 'uuid';
+
+export const timeAgo = (date) => {
```

Comment:
What type is `date` and the return type of the function?

Reply:
Also, let's a comment documenting the different return types. Explain a little bit what the function does and then the different formats it will return, it can be with bullet points.